utils/ort/src/main/kotlin/CopyrightStatementsProcessor.kt

```diff
@@ -239,6 +187,17 @@ private fun replaceYears(copyrightStatement: String): Pair<String, Set<Int>> {
  * TODO: Maybe treat URLs similar to years, e.g. entries which differ only in URLs and years can be merged.
  */
 object CopyrightStatementsProcessor {
+    data class Parts(
+        val prefix: String,
+        val years: Set<Int>,
+        val owner: String,
+        val originalStatements: List<String>
+    ) {
```

Comment:
Commit: Mind explaining the use case a bit, in particular: why can't the custom report use the process function, but needs the parts class?

Reply:
> why can't the custom report use the process function, but needs the parts class?

Because the `owner` information is lost / mangled into the statement again, which hinders grouping by owner easily.

Reply:
Would it make sense / be possible to adjust the data structure of the result of `processCopyrights` so that it works with your use case (instead of making `determineParts` public ?

Reply:
I guess I'll simply make the keys in `processedStatements` the `Parts`...

Reply:
Haven't investigated it yet,...was really more like a question

Reply:
I've taken a look at it again. The parts in the results are already merged, while exposing the function as you initially did allowed you to use determining parts independently of any merging applied. So, maybe exposing determine parts actually is simpler / more flexible.